src/platforms/react-native/migration.mdx

```diff
@@ -1,8 +1,49 @@
 ---
-title: Migrating from react-native-sentry
+title: Migration
 ---
 
-If you are upgrading from an earlier version of react-native-sentry you should unlink the package to ensure the generated code is updated to the latest version:
+## From 1.x to 2.x
+
+In `@sentry/react-native` version `2.x`, release health and native stack traces are now enabled by default.
+
+This version uses the [envelope endpoint](https://develop.sentry.dev/sdk/envelopes/).
+If you are using an on-premise installation it requires Sentry version
+`>= v20.6.0` to work. If you are using sentry.io nothing will change and
+no action is needed.
+
+### iOS/MacOS
+
+There are no immediate breaking changes for iOS/MacOS on the React Native side, just make sure you run `pod install` after the upgrade.
+
+If you use the `sentry-cocoa` SDK directly, you should follow its [migration guide for 5.x to 6.x](/platforms/apple/migration/#migrating-from-5x-to-6x).
+
+On iOS/MacOS, we now cache events in envelopes on the disk.
+We decided not to take the effort to migrate these few cached events from 5.x to 6.x into
+envelopes. Instead we remove them from the disk. This means you might lose a few cached
+events of your users when upgrading.
+
+### Android
+
+If you are on React Native <0.60, you will need to update this line in your `MainApplication.java`:
```

Comment:
```suggestion
If you are using our React Native SDK earlier than version 0.60, you will need to update this line in your `MainApplication.java`:
```

Reply:
Ah, this is actually referring to users on React Native less than 0.60 not our SDK.